src/sentry/pipeline/__init__.py

```diff
@@ -297,6 +297,11 @@ def error(self, message):
         logger.error("pipeline error", extra=extra)
         return render_to_response("sentry/pipeline-error.html", context, self.request)
 
+    def warning(self, message):
```

Comment:
@AniketDas-Tekky I feel like this method name is not descriptive enough. What do you think of `render_warning`?